File: Net21/RentalCars/DbStuff/Models/RentalCar.cs
namespace RentalCarsMinimalApi.DBContext.Models
{
    public class RentalCar: BaseModel
    {
        public string Name { get; set; }
        public string Url { get; set; }
        public string Description { get; set; }
        public int Price { get; set; }

    }
}

File: ReactRentalCars/src/types/RentalCar.ts
// Rental car model interface matching the API
export interface RentalCar {
  id: number;
  name: string;
  description: string;
  url: string;
  price: number;
}

// Form data for creating new rental car
export interface CreateRentalCarRequest {
  name: string;
  description: string;
  url: string;
  price: number;
}


